account-admin/byob.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: "Bring Your Own S3 Bucket"
```

Comment:
@umberto-sonnino @avivian nitpick but should we name the file `bring-your-own-bucket.mdx`, as that is what will be the URL address. For SEO purposes.

Reply:
Sure thing, I'll update it